pages/SignIn.js

```diff
@@ -0,0 +1,36 @@
+import styles from "../styles/Home.module.css";
+import { Form } from "react-bootstrap";
+import { Row } from "react-bootstrap";
+import { Col } from "react-bootstrap";
+import logo from "../public/logo.jpg";
+
+const SignIn = () => {
+  return (
+    <div className={styles.SignIn}>
+      <Form>
+        <Form.Group as={Row} className="mb-3" controlId="formPlaintextEmail">
+          <Form.Label column sm="2">
+            Email
+          </Form.Label>
+          <Col sm="10">
+            <Form.Control plaintext defaultValue="email@example.com" />
+          </Col>
+        </Form.Group>
+
+        <Form.Group as={Row} className="mb-3" controlId="formPlaintextPassword">
+          <Form.Label column sm="2">
+            Password
+          </Form.Label>
+          <Col sm="10">
+            <Form.Control type="password" placeholder="Password" />
+          </Col>
+        </Form.Group>
+      </Form>
+      <div>
+        <img src={logo} />
```

Comment:
<img width="268" alt="Screen Shot 2022-03-07 at 9 28 15 PM" src="https://user-images.githubusercontent.com/29217202/157154554-28ae3db5-6813-4c7c-97a9-d1eca8103c44.png">

I tried this on my mac and it is broken
you may want consider to simply have as

```jsx
<img src="/logo.jpg" alt="some alternate text here" />
```

Reply:
also, consider using 

```js
import Image from 'next/image'
```